resources/splunk_service.rb

```diff
@@ -81,7 +81,7 @@ def service_action(desired_action)
     def initialize_service
       return unless CernerSplunk::PathHelpers.ftr_pathname(install_dir).exist?
 
-      cmd = "#{command_prefix} enable boot-start#{platform_family?('windows') ? '' : " -user #{current_owner}"} --accept-license --no-prompt"
+      cmd = "#{command_prefix} enable boot-start#{platform_family?('windows') ? '' : " -user #{current_owner}"} --accept-license --answer-yes --no-prompt"
```

Comment:
If I'm agreeing to the license on Windows (per resources/splunk_install.rb), would this code ever be hit in a non-windows context?

Also does this perform the ftr steps as the correct user?

Reply:
Accepting the license shouldn't execute FTR
FTR executes as the boot-start user